Kianoosh76/IBS
# [IBS-94] Create PR Template for Standardized Pull Requests

## Description

This PR adds a pull request template to provide guidance for both PR authors and reviewers.

## Changes Made

- Created a PULL_REQUEST_TEMPLATE.md file.
- Added a checklist for both the PR author and reviewer.
- Included instructions for testing and command to run if needed.

## Test Guidelines

1. Ensure you have pulled the latest changes from the repository.
2. Create a new branch for testing.
3. Make changes to the PR template file.
4. Save the changes and commit them to your branch.
5. Push the changes to the remote repository.
6. Open a new pull request and observe the template in the PR description.

```
```

## Checklist

### For PR author:
- [ ] I have added appropriate comments to the code.
- [ ] I have ensured code style and formatting are consistent.
- [ ] I have included unit tests or integration tests where applicable.
- [ ] I have considered edge cases and handled them appropriately.
- [ ] I have celebrated my accomplishments 🎉

### For PR reviewer:
- [ ] I have reviewed the code changes.
- [ ] I have verified that comments are clear and helpful.
- [ ] I have ensured code style and formatting align with project standards.
- [ ] I have examined any included tests and their coverage.
- [ ] I have considered potential edge cases and provided feedback on them.
